python/valis/db/db.py

```diff
@@ -62,26 +67,33 @@ def connect_db(db, orm: str = 'peewee'):
 # local local dev
 # pdb.connect_from_parameters(dbname='sdss5db', host='localhost', port=6000, user='u0857802')
```

Comment:
Is this still needed locally?  If not can we remove this line?

Reply:
No, this is not needed. Deleted.